### ISSUE TITLE
No way to run fixed source calculation using Python API

### ISSUE THREAD

**Issue by paulromano** [mit-crpg/openmc#506]
If I'm not mistaken, there is no way to run a fixed source calculation using the Python API. We ought to fix that for the next release. @wbinventor Let me know your thoughts.

**Comment by wbinventor** [comment]
I've never run OpenMC in fixed source mode. What are we missing? Does the
Executor need to use a special flag at runtime, or does the API sinply need
to support an XML attribute to switch between eigenvalue and fixed source
modes?

If I'm not mistaken, there is no way to run a fixed source calculation
using the Python API. We ought to fix that for the next release. @wbinventor
<https://github.com/wbinventor> Let me know your thoughts.

—
Reply to this email directly or view it on GitHub
<https://github.com/mit-crpg/openmc/issues/506>.

**Comment by paulromano** [comment]
There is a `<fixed_source>` element that works analogous to `<eigenvalue>`. So, in the XML input you would have:
```xml
<fixed_source>
  <batches>100</batches>
  <particles>10000</particles>
</fixed_source>
```

**Comment by wbinventor** [comment]
See PR #508.

**Comment by paulromano** [comment]
Closed with merging of #508
